composer.json

```diff
@@ -19,7 +19,7 @@
   ],
   "require": {
     "php": ">=7.4",
-    "contao/core-bundle": "~4.9"
+    "contao/core-bundle": "~4.9 || ^5.0"
```

Comment:
Maybe we should drop the support for 4.9
```suggestion
    "contao/core-bundle": "^4.13 || ^5.0"
```

Reply:
You're right. I guess some changes are not supported in Contao 4.9

Reply:
yes, sorry for that. i forgot to raise the minimum version.